app/views/shared/_footer.html.slim

```diff
@@ -3,30 +3,33 @@ footer.footer.mt-2.pt-2.mb-0
     .row
       .col-lg-5.col-xs-12.about-company
         h4
-          | OSBB Application
+          = t 'footer.h4_title'
         p.pr-5.text-white-50
-          | OSBB is a legal entity created by the owners of apartments and / or non-residential
-          | premises of an apartment building for common use.
+          = t 'footer.p_description'
       .col-lg-3.col-xs-12.links
         h5.mt-lg-0.mt-sm-3
-          | Links
+          = t 'footer.h5_links'
         ul.m-0.p-0
           li
             | - 
             a[href="#"]  
-              | About us
+              = t 'footer.link_about_us'
       .col-lg-4.col-xs-12.location
         h5.mt-lg-0.mt-sm-3
-          | Location
-        p Akademika Sakharova 23z, Ivano-Frankivsk
+          = t 'footer.h5_loc'
+        p.mb-0
+          = t 'footer.p_addr'
         p.mb-0
           i.fa.fa-phone.mr-3
-          | +380-342-723-580
+          = t 'footer.p_mob_number'
         p.mb-0
           i.fa.fa-envelope.mr-3
-          | career@softserveinc.com
+          = t 'footer.p_emile'
+        p.d-flex.justify-content-between.float-right
+          = link_to 'en', url_for(locale: :en), class: 'auth-btn float-right p-0 m-0'
+          = link_to 'ua', url_for(locale: :ua), class: 'auth-btn float-right p-0 m-0'
```

Comment:
For ruby method arguments that are more then two, put them on their own line after the first argument.